src/app/features/tech-record/tech-record-routing.module.ts

```diff
@@ -15,7 +15,7 @@ const routes: Routes = [
     resolve: { load: TechRecordViewResolver }
   },
   {
-    path: ':techCreatedAt',
+    path: 'historic/:techCreatedAt',
```

Comment:
Probably out of the scope of this PR but now that the current record is the default record to be presented on `/tech-records/{systemNumber}`, if you route to provisional it would now be called 'historic' as well as if you went from the provisional back to the current. 

Initial:
![image](https://user-images.githubusercontent.com/92087051/192545763-7e606561-09c4-4008-b316-8ea7c37a6d47.png)
After going to provisional:
![image](https://user-images.githubusercontent.com/92087051/192546114-b4e4a6e1-a69b-44f7-b258-dcd1e6ef852b.png)
Back to current:
![image](https://user-images.githubusercontent.com/92087051/192546005-ce01abd6-00c8-4a41-ae2d-e9d8903b1622.png)

Reply:
This change is so the 'historic' route can be differentiated from an invalid route. Without this change, `tech-records/3506666/fhjkdsgfjhdsf` would still resolve to the historic view while we actually want it to be 'Page not found'. Having an extra segment like `historic/:createdAt` will give us better control of the routing.
The issue you described above is part of my explanation why the historic tech record view should be the end of the line [here](https://dvsa.atlassian.net/browse/CB2-5982). Only the current record should show the history table and test records. Particularly when test record are on a system number basis, we shouldn't give the impression of showing tests for a 'historic' version.